src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/all.asset.types.osx.deps.json

```diff
@@ -1,7 +1,7 @@
 {
   "runtimeTarget": {
     "name": ".NETCoreApp,Version=v1.0/osx.10.11-x64",
-    "signature": "6c0ed2851936790712df303bf7e27fc94dfbd644"
+    "signature": ""
```

Comment:
Did we discuss the hash removal with the host team to verify that there aren't scenarios in which these get validated?

Reply:
Also, is the `signature` field required in the schema?  Could we just not emit it at all?

Reply:
It was confirmed that this field is not used. I think we could omit it entirely. I will forward you both the thread.

cc @vitek-karas

Reply:
The host doesn't seem to use it. The `DependencyContext` can read/write it, but I don't think it uses it either.

Reply:
The DependencyContext APIs are writing this, so we'd need a change in that library if we wanted to not include the field in the deps file at all.